bootup.md

```diff
@@ -0,0 +1,78 @@
+# Booting sequence in the case of FullMetalUpdate : 
+
+ 1. **Primary processor steps :**
+
+     1. Primary bootstrap to initialize interrupt/exception vectors, clocks and RAM
+     1. **U-boot** is decompressed and loaded into the RAM
+     1. Execution is passed to U-boot
+
+ 1. **U-boot boot sequence :**
+
+     1. Flash, serial console, Ethernet MAC Address (etc.) are configured
+     1. A minimal bootscript called `boot.scr` is executed and is used only to load a
+        **custom** script called `uEnv.txt`, written by us. These two scripts (the minimal
+        one and the custom one) are both defined in
+        [meta-fullmetalupdate-extra][meta_fmu_e] in the dynamic-layers folder. Those are
+        both handled by the `imx-bootfiles.bb` recipe.
```

Comment:
And for those which are not based on imx6 or imx8 SOC ?

Reply:
You're correct, I should not mention imx-bootfiles.bb since it's specific to iMX.